### PR TITLE
Document überzug preview method in the manpage

### PR DESCRIPTION
**Pull request by toonn**
I fixed a typo in the kitty section. Reordered method sections alphabetically because the list keeps growing and I documented the überzug method.

### CODE REVIEW

File: doc/ranger.pod [hunk 1]
```diff
@@ -225,6 +225,15 @@ This only works in terminology. It can render vector graphics, but works only lo
 
 To enable this feature, set the option C<preview_images_method> to terminology.
 
+=head3 ueberzug
+
+Überzug is a command line utility which draws images on terminals using child
+windows. It requires PIL (or pillow) and relies on X11. This makes it
+compatible (in a limited way, i.e., tmux splits are not supported) with many
```
Comment (Vifon) [inline doc/ranger.pod]: Very minor: I don't think there should be a comma after "i.e."

File: doc/ranger.pod [hunk 2]
```diff
@@ -225,6 +225,15 @@ This only works in terminology. It can render vector graphics, but works only lo
...
+terminals and tmux but not the linux console or Wayland.
```
Comment (Vifon) [inline doc/ranger.pod]: Let's capitalize "Linux", come on.

File: doc/ranger.pod (hunk 1 above)
Comment (toonn) [reply doc/ranger.pod]: I adhere to the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html) on this.

File: doc/ranger.pod (hunk 2 above)
Comment (toonn) [reply doc/ranger.pod]: Woops, guess I didn't because I rarely capitalize cli stuff : )

File: doc/ranger.pod (hunk 1 above)
Comment (Vifon) [reply doc/ranger.pod]: TIL

**Pull request merged**